File-Formats/DocIO/Create-Word-document-in-ASP-NET-Core.md

```diff
@@ -14,15 +14,15 @@ Syncfusion Essential DocIO is a [.NET Core Word library](https://www.syncfusion.
 
 Step 1: Create a new ASP.NET Core Web application project.
 
-![Create ASP.NET Core Web application in Visual Studio](ASP-NET-Core_images/CreateProject.png)
+![Create ASP.NET Core Web application in Visual Studio with File Format DocOI](asp-net-core_images/file-format-docio-aspnet-core-web-application.png)
```

Comment:
Don't change image alt text name. Revert in all places. Previous image alt text is fine. So, revert your changes.

Change the image name like as "create-word-document-in-aspnet-core-web

Recheck and update the image name in all places.

Reply:
Made changes